mod/discussions/views/default/discussion/discussion.js

```diff
@@ -30,13 +30,21 @@ elgg.discussion.Reply.prototype = {
 
 	showForm: function () {
 		this.getForm().slideDown('medium').data('hidden', 0);
+		var $form = this.getForm();
+		$form.slideDown('medium').data('hidden', 0);
+		$('body').animate({
+			scrollTop: $form.offset().top
+		}, 500);
 	},
 
 	loadForm: function () {
 		var that = this;
-
+		var spinner = require('elgg/spinner');
```

Comment:
You can't use synchronous require outside of a module. Might work due to race condition but it won't be reliable.